Libraries/LibC/stdlib.cpp

```diff
@@ -338,10 +401,10 @@ char* mkdtemp(char* pattern)
         struct stat st;
         int rc = lstat(pattern, &st);
         if (rc < 0 && errno == ENOENT) {
-	    if (mkdir(pattern, 0700) < 0)
+            if (mkdir(pattern, 0700) < 0)
                 return nullptr;
-	    return pattern;
-	}
+            return pattern;
+        }
```

Comment:
@awesomekling side effect of `clang-format` sorry

Reply:
np lol